feature/platform/integrated_circuit/otg_tests/utilization/README.md

```diff
@@ -0,0 +1,39 @@
+# IC-1: Integrated Circuit Utilization and Thresholds
+
+## Summary
+
+Test resource utilization and threshold for INTEGRATED_CIRCUIT (IC) components.
+
+## Procedure
+* IC-1.1 - Get IC component names and verify resource utilization and threshold leaves exist
+* IC-1.2 - Verify IC threshold leaves exist
+* IC-1.3 - Configure /system/resource/threshold at 10% for resource IC
+* IC-1.4 - Add routes and traffic and verify 'used' leaf increases in value
+* IC-1.5 - Add more routes than threshold-upper and verify used-threshold-upper-exceeded upper is true
+
+## Config Parameter Coverage
+/components/component/chassis/utilization/resources/resource/config/name
+/system/utilization/resources/resource/config/name/used-threshold-upper
```

Comment:
1.
By System you mean
- Chassis
- IC
- Linecard
Should we also state what we mean by system so there are no back and forth?

2.
/system/utilization/resources/resource/config/name/used-threshold-upper
Is this included in go/wbb-oc-paths? and same for `used-threshold-upper-clear`

3. What does the following path mean in context of this test?
/system/utilization/resources/resource/

Reply:
/system/utilization is where one configures a global/system-wide threshold for a given resource name.  The idea is to configure a system global threshold for INTEGRATED_CIRCUIT utilization.  (versus configuring the threshold over and over again for each INTEGRATED_CIRCUIT component

Reply:
Regarding the meaning of [/system/utilization/resources/resource](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-system.html#system-utilization-resources-resource), I think it should be done in the openconfig yang definition.  Please review and if not clear enough, raise an issue.  (I do think there is room for improvement, but we don't want to create the clear definition in the readme for this test while the yang is unclear). :)